Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt

```diff
@@ -0,0 +1,44 @@
+In the previous videos of JSON Schema, we have shown the features and basic implementations of JSON Schema. In this video, we will deliver the actual validation process with libraries and examples.
+
+Throughout this video, we will mainly focus on validation with the JavaScript library AJV and the Python library          jsonschema, but there are many other libraries in different programming languages and tools to validate JSON Schema. 
+
+Here are some of the popular ones in different programming languages.  There are many more validators. For further information, you can check the JSON schema website.
+
+This  is the basic validation flow that all libraries follow.
+The library takes a schema for your JSON data and validates your data according to the schema. Outputs the validation result as a boolean 
+
+So let's start with AJV! It is a JavaScript library.
+
+This is the same temperature sensor example we used before. Let’s examine this in JavaScript code.
+
+First, We are importing the AJV. 
+
+This is the Schema that describes the JSON. 
+
+With this line of code, AJV takes schema as a parameter compiles it to function, and caches it.
+
+This is the data to be validated. 
+
+This step actually validates the data and writes the result into a valid boolean. 
+
+This step is for printing the result of the validation. If valid is true, then it prints "JSON Schema is valid". Otherwise, it prints the "JSON Schema is not valid." and the error message. 
```

Comment:
We should change the slide here and not use "valid" as the variable name, it sounds wrong. Probably better to use `validationResult`

Reply:
Additionally, we are not validating JSON Schema but JSON. So it is wrong to say "JSON Schema is valid". I think we should say "JSON is valid according to Schema"